python/ray/data/_internal/execution/operators/actor_pool_map_operator.py

```diff
@@ -292,7 +297,11 @@ def current_processor_usage(self) -> ExecutionResources:
         )
 
     def pending_processor_usage(self) -> ExecutionResources:
-        num_pending_workers = self._actor_pool.num_pending_actors()
+        # Both pending and restarting actors count towards pending processor usage
+        num_pending_workers = (
+            self._actor_pool.num_pending_actors()
+            + self._actor_pool.num_restarting_actors()
+        )
```

Comment:
nit: Don't need to do in this PR, but we'd probably revisit our terminology to avoid confusion -- since we're not folding restarting actor usage into this method we'd probably rename it along the lines of active/inactive processor usage